examples/tree_2d_dgsem/elixir_navierstokes_taylor_green_vortex_sutherland.jl

```diff
@@ -0,0 +1,91 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Navier-Stokes equations
+
+# TODO: parabolic; unify names of these accessor functions
+prandtl_number() = 0.72
+
+# Use Sutherland's law for a temperature-dependent viscosity
+@inline function mu(u, equations)
```

Comment:
Just a thought: If this is used more often, we could create something like
```julia
struct SutherlandsLaw{RealT<:Number}
    T_ref::RealT
    R_specific::RealT
    C::RealT
    mu0::RealT
end

function (sutherlands_law::SutherlandsLaw)(u, equations::Union{...})
    @unpack T_ref, R_specific, C, mu0 = sutherlands_law
    ...
    reuturn mu0 * (T_ref + C) / (T + C) * (T / T_ref)^1.5
end
```
And then define (and possibly export) some commonly used ones, e.g.,
```julia
const mu_dry_air = SutherlandsLaw(291.15, 287.052874, ...)
```
But that's for another PR 🙂

Reply:
True, but then we should also decide if we keep this version or the extended one found in [FLUXO](https://github.com/project-fluxo/fluxo/blob/c7e0cc9b7fd4569dcab67bbb6e5a25c0a84859f1/src/equation/navierstokes/equation_vars.f90#L654-L669)